nursery/connect-to-wmi-namespace-via-wbemlocator.yml

```diff
@@ -0,0 +1,17 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: connect to WMI namespace via WbemLocator
+    namespace: collection/database/wmi
```

Comment:
this is not necessarily `collection`

Reply:
very true - maybe add to `host-interaction/wmi` instead?